mlflow/server/js/src/experiment-tracking/components/ExperimentListView.tsx

```diff
@@ -119,6 +121,27 @@ export const ExperimentListView = ({
                 />
               </Button>
             </Tooltip>
+            <Tooltip
+              componentId="mlflow.experiment_list_view.bulk_delete_button_tooltip"
+              content={
+                <FormattedMessage
+                  defaultMessage="Select experiments from the table to be deleted"
+                  description="Experiments page delete experiments button tooltip message"
+                />
+              }
+            >
+              <Button
+                componentId="mlflow.experiment_list_view.bulk_delete_button"
+                onClick={() => setShowBulkDeleteExperimentModal(true)}
+                data-testid="delete-experiments-button"
+                disabled={checkedKeys.length < 1}
```

Comment:
could we make it a danger button? i realize it's kind of inconsistent in the app today but we should standardize that delete buttons are `secondary` and `danger` styled 
```suggestion
                disabled={checkedKeys.length < 1}
                danger
```